### PR TITLE
bump pmdsky-debug

### PR DESCRIPTION
**Pull request by theCapypara**
Some symbols were renamed, this still needs to be fixed, and other SkyTemple sources must be checked as well.

### CODE REVIEW

File: skytemple_files/common/ppmdu_config/pmdsky_debug/extras.py [hunk 1]
```diff
@@ -106,7 +106,12 @@ class ExtraAllSymbolsProtocol(Protocol):
 class ExtraEuArm9Functions:
     # TODO: Is this really the same as DebugPrint0 for EU???
     DebugPrint2 = Symbol(
-        [0x200C284 - EuArm9Section.loadaddress], [0x200C284], None, DEBUG_PRINT2_DESC
+        [0x200C284 - EuArm9Section.loadaddress],
+        [0x200C284],
+        None,
+        "DebugPrint2",
+        DEBUG_PRINT2_DESC,
+        "",
```
Comment (End45) [inline skytemple_files/common/ppmdu_config/pmdsky_debug/extras.py:114]:
```suggestion
        None,
```
As stated in the description of `Symbol`: "C type of this symbol. Empty string if unknown. None for functions."

File: skytemple_files/common/ppmdu_config/pmdsky_debug/extras.py [hunk 2]
```diff
@@ -192,7 +205,12 @@ class ExtraEuSections:
 
 class ExtraNaArm9Functions:
     DebugPrint2 = Symbol(
-        [0x200C30C - NaArm9Section.loadaddress], [0x200C30C], None, DEBUG_PRINT2_DESC
+        [0x200C30C - NaArm9Section.loadaddress],
+        [0x200C30C],
+        None,
+        "DebugPrint2",
+        DEBUG_PRINT2_DESC,
+        "",
```
Comment (End45) [inline skytemple_files/common/ppmdu_config/pmdsky_debug/extras.py:213]:
```suggestion
        None,
```
See above.

File: skytemple_files/common/ppmdu_config/pmdsky_debug/extras.py [hunk 3]
```diff
@@ -277,27 +303,33 @@ class ExtraNaSections:
 
 
 class ExtraJpArm9Functions:
-    DebugPrint2 = Symbol(None, None, None, DEBUG_PRINT2_DESC)
+    DebugPrint2 = Symbol(None, None, None, "DebugPrint2", DEBUG_PRINT2_DESC, "")
```
Comment (End45) [inline skytemple_files/common/ppmdu_config/pmdsky_debug/extras.py:306]:
```suggestion
    DebugPrint2 = Symbol(None, None, None, "DebugPrint2", DEBUG_PRINT2_DESC, None)
```
See above.

**Review by End45 — CHANGES_REQUESTED** [review]

**Comment by theCapypara** [comment]
Thanks

**Review by End45 — APPROVED** [review]

**Pull request merged**